lib/ts/recipe/emailpassword/components/features/emailVerification/emailVerification.tsx

```diff
@@ -128,7 +151,10 @@ class EmailVerification extends PureComponent<FeatureBaseProps, { token: string
                     action: EMAIL_PASSWORD_SUCCESS_ACTION.VERIFY_EMAIL_SENT
                 }),
             onEmailAlreadyVerified: () =>
-                this.getRecipeInstanceOrThrow().redirect({ action: "SUCCESS" }, this.props.history)
+                this.getRecipeInstanceOrThrow().redirect(
+                    { action: EMAIL_PASSWORD_REDIRECTION_URL_ACTION.SUCCESS },
```

Comment:
I prefer not using constants like `EMAIL_PASSWORD_REDIRECTION_URL_ACTION.SUCCESS`. Adds more boilerplate coding etc.. instead make action have a specific type based on string and use that everywhere.. is easier to read, code etc.